tests/e2e/statefulsets.go

```diff
@@ -116,13 +116,18 @@ var _ = ginkgo.Describe("statefulset", func() {
 			ginkgo.By("CNS_TEST: Running for vanilla k8s setup")
 			scParameters = nil
 			storageClassName = "nginx-sc-default"
+			scSpec := getVSphereStorageClassSpec(storageClassName, scParameters, nil, "", "", false)
+			sc, err := client.StorageV1().StorageClasses().Create(ctx, scSpec, metav1.CreateOptions{})
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			defer func() {
+				err := client.StorageV1().StorageClasses().Delete(ctx, sc.Name, *metav1.NewDeleteOptions(0))
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
Can you use the random name generation logic here to create unique policy

Reply:
existing code for vanilla uses defaultNginxStorageClassName , I am keeping the same
Only in case of WCP i have changed to sared-ds-policy